addons/l10n_tr_nilvera/models/res_partner.py

```diff
@@ -1,8 +1,9 @@
 import logging
 import urllib.parse
 
-from odoo import api, fields, models, modules
-from odoo.exceptions import UserError
+from collections import defaultdict
+
+from odoo import api, fields, models, _
```

Comment:
order :smile:  https://www.odoo.com/documentation/17.0/contributing/development/coding_guidelines.html#imports
```suggestion
from odoo import _, api, fields, models
```